tests/mock_tables/asic1/config_db.json

```diff
@@ -197,5 +197,16 @@
         "holdtime": "0",
         "asn": "65100",
         "keepalive": "0"
```

Comment:
this profile should be for lossless not lossy

Reply:
The comment is outdated, but to address it, the test vectors are added as a dummy to validate the functionality.